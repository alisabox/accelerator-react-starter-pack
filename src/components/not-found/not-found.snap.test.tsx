import { render } from '@testing-library/react';
import NotFound from './not-found';

describe('Component: Not Found', () => {
  test('should render correctly vs. snapshot', () => {
    const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
  });
});
