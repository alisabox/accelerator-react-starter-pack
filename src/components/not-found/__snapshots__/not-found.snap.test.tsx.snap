// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Not Found should render correctly vs. snapshot 1`] = `
<div>
  <div
    class="not-found"
  >
    <header
      class="header"
      id="header"
    >
      <div
        class="container header__wrapper"
      >
        <a
          class="header__logo logo"
          href="/#"
        >
          <img
            alt="Логотип"
            class="logo__img"
            height="70"
            src="./img/svg/logo.svg"
            width="70"
          />
        </a>
      </div>
    </header>
    <main
      class="page-content"
    >
      <div
        class="container"
      >
        <h1
          class="page-content__title title title--not-found "
        >
          404 Not Found
        </h1>
      </div>
    </main>
  </div>
</div>
`;
